scipy-1.0/paper.tex

```diff
@@ -878,11 +878,19 @@ \subsection*{Test and benchmark suite}
 \subsubsection*{Test suite}
 
 Test-driven development has been described as a way to manage fear and
-uncertainity when making code changes\cite{TDD-Beck}. For each component of SciPy, we write multiple small executable
-tests that verify its intended behavior. The collection of these, known as a \emph{test suite}, increases confidence in
-the correctness and accuracy of the library, and lets us make code modifications known not to alter desired behavior. Monitoring the number of lines
-of code in SciPy covered by unit tests is one way we maintain
+uncertainity when making code changes\cite{TDD-Beck}. For each component of 
+SciPy, we write multiple small executable tests that verify its intended
+behavior. The collection of these, known as a \emph{test suite}, increases
+confidence in the correctness and accuracy of the library, and lets us make
+code modifications known not to alter desired behavior.
+According to the practice of continuous integration, all proposed contributions
+to SciPy are temporarily integrated with the master branch of the library
+before the test suite is run, and all tests must pass before the contribution
+is permanently merged.
+Continuously monitoring the number
```

Comment:
"In formal contexts, continually should be used to mean “very often; at regular or frequent intervals,” and continuously to mean “unceasingly; constantly; without interruption."
So not a perfect use of the word; added as an attempt to tie the preceeding sentence in with the rest.